proxies/dotnet/Controllers/Client.cs

```diff
@@ -0,0 +1,97 @@
+using Microsoft.AspNetCore.Mvc;
+using DevCycle.SDK.Server.Cloud.Api;
+using DevCycle.SDK.Server.Local.Api;
+using DevCycle.SDK.Server.Common.Model.Local;
+using DevCycle.SDK.Server.Common.Model.Cloud;
+using System.Text.Json.Serialization;
+
+namespace dotnet.Controllers;
+
+public class ClientOptions: DVCLocalOptions
+    {
+        [JsonPropertyName("configPollingIntervalMs")]
+        new public int ConfigPollingIntervalMs { get; set; }
+
+        [JsonPropertyName("eventFlushIntervalMS")]
+        new public int EventFlushIntervalMs { get; set; }
+
+        [JsonPropertyName("baseURLOverride")]
+        public string? BaseURLOverride { get; set; }
+
+        [JsonPropertyName("enableEdgeDB")]
+        public bool? EnableEdgeDB { get; set; }
+    }
+
+public class ClientRequestBody {
+    public string? clientId { get; set;}
```

Comment:
Pascal case for properties?

Reply:
I could but then I'd have to specify the real name with the JSON property override and this just seemed easier 😬

Reply:
it's a little odd that you've defined the ClientOptions using `JsonPropertyName` but not the top level RequestBody

Reply:
thats partly cuz the ClientOptions need to match the interface of the SDK options object

Reply:
I can change these for consistency